services/ui-src/src/components/sections/login/LocalLogins.jsx

```diff
@@ -1,9 +1,14 @@
-import React from "react";
-import { useNavigate } from "react-router-dom";
-import { useFormFields } from "../../../hooks/useFormFields";
+import React, { useState } from "react";
 import { loginUser } from "../../../util/apiLib";
+//components
+import { AlertNotification } from "../../alerts/AlertNotification";
+//utils
+import { useFormFields } from "../../../hooks/useFormFields";
+import { loginError } from "../../../verbiage/errors";
+import { useNavigate } from "react-router-dom";
```

Comment:
nit: sorry I should have been clear.
Move the `useNavigate` import to line 2, and bring `loginUser` back to below the utils comment
```suggestion
import React, { useState } from "react";
import { useNavigate } from "react-router-dom";
// components
import { AlertNotification } from "../../alerts/AlertNotification";
// utils
import { loginError } from "../../../verbiage/errors";
import { loginUser } from "../../../util/apiLib";
import { useFormFields } from "../../../hooks/useFormFields";
```